src/main/frontend/components/page.cljs

```diff
@@ -308,6 +342,14 @@
         (when (and (not sidebar?)
                    (not block?))
           [:div.flex.flex-row.space-between
+           (when (mobile-util/is-native-platform?)
```

Comment:
Better make it available on browsers(narrow screens) as well.

Reply:
Will add `is-mobile` check here.